SUMMARY.md

```diff
@@ -2,6 +2,9 @@
 
 * [Technische Spezifikation](README.md)
 
+## PYTHON DEPENDENCIES
```

Comment:
Warum in all-caps? `Python dependencies` reicht

Reply:
Ich bin der Meinung es sollte "Python Dependencies" heißen, analog zu den anderen Überschriften.